.github/workflows/release.yml

```diff
@@ -172,12 +187,22 @@ jobs:
             echo "### 🧪 Dry Run Completed" >> $GITHUB_STEP_SUMMARY
             echo "- Version bump: **${{ github.event.inputs.version_type }}**" >> $GITHUB_STEP_SUMMARY
             echo "- Branch: **${{ github.event.inputs.branch }}**" >> $GITHUB_STEP_SUMMARY
+            if [[ "${{ github.event.inputs.version_type }}" == pre* ]]; then
+              echo "- npm tag: **${{ github.event.inputs.prerelease_tag }}**" >> $GITHUB_STEP_SUMMARY
+            else
+              echo "- npm tag: **latest**" >> $GITHUB_STEP_SUMMARY
+            fi
             echo "- Status: **Dry run - no changes made**" >> $GITHUB_STEP_SUMMARY
           else
             echo "### ✅ Release Completed" >> $GITHUB_STEP_SUMMARY
             echo "- Version bump: **${{ github.event.inputs.version_type }}**" >> $GITHUB_STEP_SUMMARY
             echo "- Branch: **${{ github.event.inputs.branch }}**" >> $GITHUB_STEP_SUMMARY
             echo "- New tag: **$NEW_TAG**" >> $GITHUB_STEP_SUMMARY
+            if [[ "${{ github.event.inputs.version_type }}" == pre* ]]; then
+              echo "- npm tag: **${{ github.event.inputs.prerelease_tag }}**" >> $GITHUB_STEP_SUMMARY
+            else
+              echo "- npm tag: **latest**" >> $GITHUB_STEP_SUMMARY
+            fi
             echo "- Packages published to npm" >> $GITHUB_STEP_SUMMARY
             echo "- Draft GitHub release created" >> $GITHUB_STEP_SUMMARY
```

Comment:
add a link to `Draft GitHub release`, let user can access it and more easy to publish the draft